LandSandBoat/server
# [Lua, Fishing, Quest] The Sand Charm converted to IF format

**_I affirm:_**

## What does this pull request do?

Converts the quest "The Sand Charm" to IF format.

## Steps to test these changes

!exec p = GetPlayerByName("You")
!exec p:setFame(2, 224)
Talk to blandine, quest won't start
!exec p:setFame(2, 225) -- level 4 windurst/Mhaura fame
Talk to blandine in dock area: quest won't start
Talk to blandine in town area: quest will start
Talk to Zexu, talk to blandine again, cs with Celestina
Talk to celestina, quest should be in quest log now
!additem 13095 -- sand charm
Trade sand charm to celestina: quest complete, get rewards
Talk to blandine again for a one time flavor text
